scripts/deployment/vertex_ai_phase4_automation.py

```
@@ -166,11 +166,11 @@ def _check_iam_permissions(self) -> bool:
         ]
 
         try:
-            result = subprocess.run(['gcloud', 'projects', 'get-iam-policy', self.config.project_id, 
-                                   '--flatten=bindings[].members', 
-                                   '--format=value(bindings.role)'], 
+            result = subprocess.run(['gcloud', 'projects', 'get-iam-policy', self.config.project_id,
+                                   '--flatten=bindings[].members',
+                                   '--format=value(bindings.role)'],
                                   capture_output=True, text=True, check=True)
-            user_email = subprocess.run(['gcloud', 'config', 'get-value', 'account'], 
+            user_email = subprocess.run(['gcloud', 'config', 'get-value', 'account'],
```

Ruff (0.12.2)

Local variable `user_email` is assigned to but never used

Remove assignment to unused variable `user_email`

(F841)